.github/pull_request_template_.md

```diff
@@ -0,0 +1,53 @@
+<!-- markdownlint-disable MD041 -->
```

Comment:
Might be worth checking if we can remove this comment as it may confuse contributors. I can do this separately, let’s get the first version in 🙂